### PR TITLE
Content block with cards content model

### PR DESCRIPTION
**Pull request by mfearer94**
I've added the content model to the content block with cards as well as an anchor link to get to content guidance. I also added extra security to the Box link that already exists on this page.

### Changelog

**New**

- Content model
- Anchor link

**Changed**

- Extra security for Box link

### CODE REVIEW

File: src/pages/components/content-block-with-cards.mdx [hunk 1]
```diff
@@ -37,9 +38,22 @@ Sometimes a video will support the pages' narrative, but should not be prominent
 
 ![Image of Content block simple with aside elements](../../images/pattern/content-block-with-cards/content-block-with-cards-with-videos.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) | Notes | 
+| -------- | --------- | ---------- | ---------- | ----------| ----------|
+| Headline | Short copy | Yes | 1 | XXS (40 / 55)| - |
+| TOC headline override | Short copy | No | 1 | Mini (25 / 35)| - |
```
Comment (photodow) [inline src/pages/components/content-block-with-cards.mdx:46]: 😅

File: src/pages/components/content-block-with-cards.mdx [hunk 2]
```diff
@@ -37,9 +38,22 @@ Sometimes a video will support the pages' narrative, but should not be prominent
...
+| [Card](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/card) | Component | Yes | 2-6 | - | You can use the simple card, image card, or video card, but all cards must be of the same type. |
+| [CTA (card)](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/cta) | Component | No | 1 | - | - |
```
Comment (photodow) [inline src/pages/components/content-block-with-cards.mdx:48]: Calling out content block for what it's worth.

File: src/pages/components/content-block-with-cards.mdx [hunk 3]
```diff
@@ -37,9 +38,22 @@ Sometimes a video will support the pages' narrative, but should not be prominent
...
+
+Note: This uses the underlying component "Card group". You can find more information about that component <a href="https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/card-group">here</a>.
```
Comment (photodow) [inline src/pages/components/content-block-with-cards.mdx:50]: Is there a reason we call it out here instead of up above? Just curious...

Comment (mfearer94) [reply src/pages/components/content-block-with-cards.mdx:50]: I'm updating this to just be card group

**Comment by mfearer94** [comment]
@photodow  Need review on this one.

**Comment by mfearer94** [comment]
@JennySanchez This is ready for your review

**Review by JennySanchez — APPROVED** [review]
Thank you, Michael!

**Pull request merged**